utils/table_printer_windows.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	"os"
+)
+
+func (t *ttyTablePrinter) availableWidth() int {
+	if os.Getenv("WT_SESSION") == "" {
```

Comment:
As mentioned above, WT defines this. It's not specific to powershell.exe which, if run directly, doesn't run in WT (though it may in Windows 11 if WT is set as the default terminal).

cmd.exe, powershell.exe, and pwsh.exe by default run in conhost.exe, and this problem - which I can repro - appears to happen only there (though, to be frank, I don't have anything other than conhost and WT installed). But with WT, I've never seen this.

So given all that, the check for `WT_SESSION` for truecolor should be platform agnostic, but if you wanted to solve this problem here, perhaps check that the environment `windir` is defined byt not `WT_SESSION`. Short of getting the current process name - certainly another option - that would probably be easiest.

Reply:
Sorry, I misread this as it was checking for empty. This is probably fine then. Checking for `windir` in this case would be pointless. If there are any other terminals people are running in Windows they are only losing a single block char. Anything popular enough could always be added to this.